File: CS/07_Security/RemoveAllDigitalSignatures.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Security.Cryptography.X509Certificates;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace RemoveAllDigitalSignatures
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Load a ppt document
            Presentation ppt = new Presentation();
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\RemoveAllDigitalSignatures.pptx");

            //Remove all digital signatures
            if (ppt.IsDigitallySigned == true)
            {
                ppt.RemoveAllDigitalSignatures();
            }
            //Save the document
            string output = @"RemoveAllDigitalSignatures_result.pptx";
            ppt.SaveToFile(output, FileFormat.Pptx2010);
            //Launch the file
            OutputViewer(output);
        }
        private void OutputViewer(string filename)
        {
            try
            {
                System.Diagnostics.Process.Start(filename);
            }
            catch { }
        }
    }
}

File: VB/07_Security/RemoveAllDigitalSignatures.vb
Imports System.ComponentModel
Imports System.Security.Cryptography.X509Certificates
Imports System.Text
Imports Spire.Presentation

Namespace RemoveAllDigitalSignatures
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Load a ppt document
			Dim ppt As New Presentation()
			ppt.LoadFromFile("..\..\..\..\..\..\Data\RemoveAllDigitalSignatures.pptx")

			'Remove all digital signatures
			If ppt.IsDigitallySigned = True Then
				ppt.RemoveAllDigitalSignatures()
			End If
			'Save the document
			Dim output As String = "RemoveAllDigitalSignatures_result.pptx"
			ppt.SaveToFile(output, FileFormat.Pptx2010)
			'Launch the file
			OutputViewer(output)
		End Sub
		Private Sub OutputViewer(ByVal filename As String)
			Try
				Process.Start(filename)
			Catch
			End Try
		End Sub
	End Class
End Namespace